src/browser.ts

```diff
@@ -70,7 +70,7 @@ export const launchBrowser = async (
       `--proxy-server=http://localhost:${appSettings.proxy.port}`,
       `--ignore-certificate-errors-spki-list=${certificateSPKI}`,
       disableChromeOptimizations,
-      url ?? '',
+      url ?? 'about:blank',
```

Comment:
I'm still able to reproduce the issue on Windows. My guess `url` is an empty string and doesn't trigger nullish operator fallback, the `||` operator should do the trick. 

You can also verify this on macos: about:blank should open as an empty page, currently it opens to chrome homepage with google search.

Reply:
Might be worth to also to `url.trim()` to handle the user entering a bunch of spaces.

Reply:
You're right, I didn't even notice this on MacOS 🤦‍♂️